tests/sles4sap/hana_install.pm

```diff
@@ -105,8 +99,14 @@ sub run {
     my $cmd = join(' ', $hdblcm, @hdblcm_args);
     assert_script_run $cmd, $tout;
 
-    upload_install_log;
+    # Enable autostart of HANA HDB, otherwise DB will be down after the next reboot
+    my $hostname = script_output 'hostname';
+    file_content_replace("$mountpts{hanashared}->{mountpt}/${sid}/profile/${sid}_HDB${instid}_${hostname}", '^Autostart[[:blank:]]*=.*' => 'Autostart = 1');
```

Comment:
Did you test this with the HanaSR tests? This setting could impact that scenario where HANA is restarted by the cluster, but I'm not 100% sure.

Reply:
You're right, this shouldn't be done in case of cluster.